lib/dat-worker-pool/runner.rb

```diff
@@ -40,6 +42,8 @@ def start
     # non-timeout errors will be re-raised so they can be caught and handled (or
     # shown when the process exits)
     def shutdown(timeout = nil, backtrace = nil)
+      timeout_message = timeout ? "in #{timeout} second(s)" : "without a timeout"
```

Comment:
@jcredding I would change the "in X second(s)" to "(timeout: X second(s)".  The "in X seconds" makes me think that the shutdown won't happen until X seconds have passed (makes it sound more like a *delayed* shutdown).

This may mean you should also change the "without a timeout" to "(timeout: none)" or something to better match.  I'm open to other alternatives too if you can think of one (I know what I'm suggesting is a little robotic but this is debug stuff so :shrug:).